cypress/integration/channel/autocomplete_shown_each_channel_spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Identical Message Drafts', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('M14432 shows Autocomplete in each channel', () => {
+        // 1. Go to test Channel A on sidebar
+        cy.get('#sidebarItem_town-square').scrollIntoView();
+        cy.get('#sidebarItem_town-square').should('be.visible').click();
+
+        // * Validate if the channel has been opened
+        cy.url().should('include', '/ad-1/channels/town-square');
+
+        // 2. Start a draft in Channel A containing just "@"
+        cy.get('#post_textbox').scrollIntoView();
+        cy.get('#post_textbox').clear().type('@');
```

Comment:
2 lines above should just be `cy.get('#post_textbox').type('@');`.
- no need to scroll into view since it's always expected to be in a view.
- clearing should be handled in `before` hook

Same comment for channel B.